src/pkg/services/m365/api/lists.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/alcionai/corso/src/pkg/services/m365/api/graph"
 )
 
-var ErrCannotCreateWebTemplateExtension = clues.New("unable to create webTemplateExtension type lists")
+var ErrCannotCreateNonRestorableListTemplate = clues.New("unable to create lists with non-restorable template")
```

Comment:
nit: This could be simplified to `ErrNonRestorableListTemplate`, to avoid the double negatives.